driver-core/src/main/com/mongodb/connection/CommandMessage.java

```diff
@@ -38,8 +38,12 @@
 import java.util.Map;
```

Comment:
Two substantive changes notes below.  The rest is just refactoring to clarify intent.

Reply:
ACK